src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -535,8 +535,8 @@ std::string GrpcLb::Serverlist::AsText() const {
       grpc_resolved_address addr;
       ParseServer(server, &addr);
       auto addr_str = grpc_sockaddr_to_string(&addr, false);
-      ipport = std::string(addr_str.ok() ? addr_str.value()
-                                         : addr_str.status().ToString());
+      ipport = addr_str.ok() ? addr_str->c_str()
```

Comment:
There is no need to call `c_str()` either here or on the next line, since you are assigning to a `std::string`, not a `char*`.